nilearn/plotting/displays/_slicers.py

```diff
@@ -398,16 +398,6 @@ def _map_show(
             img = reorder_img(img, resample=resampling_interpolation)
         threshold = float(threshold) if threshold is not None else None
 
-        if threshold is not None:
-            data = _safe_get_data(img, ensure_finite=True)
-            if threshold == 0:
-                data = np.ma.masked_equal(data, 0, copy=False)
-            else:
-                data = np.ma.masked_inside(
-                    data, -threshold, threshold, copy=False
-                )
-            img = new_img_like(img, data, img.affine)
-
```

Comment:
- removing this causes a regression https://github.com/nilearn/nilearn/issues/4255